prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -31,24 +31,23 @@ public class OrganizationService : BaseService, IOrganizationService
 
         public async Task<IEnumerable<Organization>> GetOrganizationsAsync(int? partyId = null)
         {
-            IQueryable<Organization> query = this.GetBaseOrganizationQuery();
-
-            if (partyId != null)
-            {
-                query = query.Where(o => o.SigningAuthorityId == partyId);
-            }
-
-            return await query
-                .Include(o => o.Sites).ThenInclude(s => s.SiteVendors)
-                .Include(o => o.Sites).ThenInclude(s => s.PhysicalAddress)
-                .Include(o => o.Sites).ThenInclude(s => s.Adjudicator)
-                .Include(o => o.Sites).ThenInclude(s => s.RemoteUsers)
+            return await GetBaseOrganizationQuery()
+                .Include(o => o.Sites)
+                    .ThenInclude(s => s.SiteVendors)
+                .Include(o => o.Sites)
+                    .ThenInclude(s => s.PhysicalAddress)
+                .Include(o => o.Sites)
+                    .ThenInclude(s => s.Adjudicator)
+                .Include(o => o.Sites)
+                    .ThenInclude(s => s.RemoteUsers)
+                .If(partyId != null, q => q.Where(o => o.SigningAuthorityId == partyId))
                 .ToListAsync();
         }
 
         public async Task<Organization> GetOrganizationAsync(int organizationId)
         {
-            return await this.GetBaseOrganizationQuery()
+            return await GetBaseOrganizationQuery()
+                .Include(o => o.Sites)
```

Comment:
Arent Sites included already in the BaseOrganizationQuery?

Reply:
No, it contains agreements, signed agreement, physical and mailing address.